microsoft-edge/toc.yml

```diff
@@ -779,123 +779,195 @@
 # =============================================================================
     - name: WebView2
       items:
+        # keep at top level?  move up or down?
         - name: Introduction to Microsoft Edge WebView2
           href: webview2/index.md
+          displayName:  # add alt lookup terms
 
+
+        # keep at top level?  move up or down?
+      # - name: Set up dev tools and create the initial app  # a more specific title
         - name: Get started
           items:
-            - name: Get started with WebView2
+            - name: Get started with WebView2  # platform-independent
               href: webview2/get-started/get-started.md
+              displayName:  # add alt lookup terms
 
             - name: Get started with WebView2 in Win32 apps
               href: webview2/get-started/win32.md
+              displayName:  # add alt lookup terms
 
             - name: Get started with WebView2 in WinForms apps
               href: webview2/get-started/winforms.md
+              displayName:  # add alt lookup terms
 
             - name: Get started with WebView2 in WinUI 2 (UWP) apps (public preview)
               href: webview2/get-started/winui2.md
+              displayName:  # add alt lookup terms
 
             - name: Get started with WebView2 in WinUI 3 (Windows App SDK)
               href: webview2/get-started/winui.md
+              displayName:  # add alt lookup terms
 
             - name: Get started with WebView2 in WPF apps
               href: webview2/get-started/wpf.md
+              displayName:  # add alt lookup terms
+
 
+        # keep at top level?  move up or down?
         - name: WebView2 sample code
           href: webview2/code-samples-links.md
 
-        - name: Concepts and How to
-          items:
-            - name: Automate and test WebView2 apps with Microsoft Edge Driver
-              href: webview2/how-to/webdriver.md
 
-            - name: Get started debugging WebView2 apps
-              href: webview2/how-to/debug.md
+        # re-order the below articles in order of the development workflow
+        # when discover a cluster of related articles, nest that set/group
+        # favor a flat toc, with more articles per bucket
```

Comment:
Important: Avoid deep nesting.
Not important: It's ok to have a bucket/parent that has a small or large number of direct children.